Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -218,7 +219,6 @@ private int GetStatusCode(Exception ex)
                 case FileNotFoundException _:
                 case ResourceNotFoundException _: return 404;
                 case MethodNotAllowedException _: return 405;
-                case RemoteServiceUnavailableException _: return 502;
```

Comment:
why?

Reply:
Because I removed that exception, it wasn't used.